std_msgs/Header header
LineSeg[] line_segments

#### Usage
#
# line_segments[index].start = Point(x1, y1, 0.)
# line_segments[index].end = Point(x2, y2, 0.)
#
#
#
#
